projects/convenience.md

```diff
@@ -0,0 +1,91 @@
+# Convenience Project
```

Comment:
```suggestion
# Developer Experience Project
```

Reply:
I guess we'll want to rename this file as well to devex or developer-experience? I left the others in the Background section because it's a discussion of history which explicitly referred to this as Convenience.

Reply:
Can we make sure that the name of this proposal can clearly be distinguished from the "Contributor Experience" proposal, since contributors to the otel project (often) also are developer or see themselves as developer ?

Reply:
Came here to say the same: perhaps "instrumentation experience"?

Reply:
Developer Experience is a generally accepted term, and making it instrumentation or any other variation will diminish the impact of the initiative publicly.

Contributor Experience is a different thing and speaks directly to the target market of that.

You could say in the subtitle "working with OpenTelemetry", but changing away from Developer Experience wouldn't have the same impact we so desperately need right now.

Reply:
+1 for keeping "Developer Experience" but _if_ we can find a way to distinguish it great, but it's not a hill I am going to die on, especially since "SIG DevEx" is catchy.

Reply:
Could Contributor Experience use a different word than Experience maybe? I've not thougt of one that captures the goal of that project but I feel like there is one...